src/Mod/Draft/draftobjects/label.py

```diff
@@ -298,6 +293,23 @@ def execute(self, obj):
 DraftLabel = Label
 
 
+def get_label_types():
+    return ["Custom",
+            "Name",
+            "Label",
+            "Position",
+            "Length",
+            "Area",
+            "Volume",
+            "Tag",
+            "Material",
+            "Label + Position",
+            "Label + Length",
+            "Label + Area",
+            "Label + Volume",
+            "Label + Material"]
```

Comment:
Are these of any importance to translate? If yes and if not in this code block, then where?

Reply:
This list should not be translated. The strings are used in the code so translating the list would be problematic with the current implementation.